pkg/services/provisioning/provisioning.go

```diff
@@ -39,13 +38,36 @@ func init() {
 	})
 }
 
+// ToDo make a better instantiation of the provisioning service.
+
+// NewProvisioningServiceImpl creates a new instance of a ProvisioningService
+// without relying on facebook's inject fwk
+func NewProvisioningServiceImpl(
```

Comment:
Could I do something cleaner?

Reply:
Yes I could. By modifying every Oss Provisioner in order make it a service. As discussed [there](https://github.com/grafana/grafana-enterprise/issues/1249#issuecomment-819301398): 
But this is out of my scope. Refactoring this amount of code will be painful and error prone.
However, I could at least remove the newProvisioningServiceImpl method.